extension-style/src/main/java/com/mapbox/maps/extension/style/light/generated/Light.kt

```diff
@@ -24,7 +24,7 @@ import kotlin.collections.HashMap
 /**
  * The global light source.
  *
- * @see <a href="https://www.mapbox.com/mapbox-gl-style-spec/#light">The online documentation</a>
+ * Check the [online documentation](https://www.mapbox.com/mapbox-gl-style-spec/#light).
```

Comment:
Is this the only place to fix link format?

Reply:
will create a separate PR to fix all the documentation links.